2011-01-12  Aurelien Jarno  <aurelien@aurel32.net>

	* locales/et_EE: Change currency to euro.

2010-01-05  Aurelien Jarno  <aurelien@aurel32.net>

	* locales/et_EE: Change first_weekday and first_workday to 2.

---
 localedata/locales/et_EE |    8 ++--
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/localedata/locales/et_EE
+++ b/localedata/locales/et_EE
@@ -109,8 +109,8 @@
 END LC_CTYPE
 
 LC_MONETARY
-int_curr_symbol     "<U0045><U0045><U004B><U0020>"
-currency_symbol     "<U006B><U0072>"
+int_curr_symbol     "<U0045><U0055><U0052><U0020>"
+currency_symbol     "<U20AC>"
 mon_decimal_point   "<U002C>"
 mon_thousands_sep   "<U00A0>"
 mon_grouping        3;3
@@ -178,8 +178,8 @@
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
 week    7;19971130;4
-first_weekday 1
-first_workday 1
+first_weekday 2
+first_workday 2
 END LC_TIME
 
 LC_MESSAGES
